src/components/GameScreen/GameScreen.tsx

```diff
@@ -17,50 +22,83 @@ interface PropTypes {
 // chess.reset() Resets board
 // chess.turn() Returns current side to move (w, b)
 
-const GameScreen = ({ id }: PropTypes) => {
+const GameScreen = ({ gameId, userKey, userName }: PropTypes) => {
+  const [chess] = useState<any>(
+    new Chess("rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1")
+  )
+  const [fen, setFen] = useState(chess.fen())
+  useEffect(() => {
+    console.log(gameId)
+    const cable = actioncable.createConsumer(`${API_WS_ROOT}`)
+    console.log('API_KEY', userKey)
+    cable.subscriptions.create({
+      channel: 'FriendlyGamesChannel',
+      api_key: userKey, 
+      extension: gameId
+    },{
+      connected: ()=> {
+        console.log('connected!')
```

Comment:
So glad we got the random disconnection problem solved (by using a different actioncable package). Good job @CJSpohn !